sympy/matrices/tests/test_eigen.py

```diff
@@ -225,6 +225,18 @@ def test_eigenvects():
         assert M*vec_list[0] == val*vec_list[0]
 
 
+def test_eigenvects_algebraic_field():
```

Comment:
This test should be marked as `@slow` because it takes more than a couple of seconds.